bundler/spec/dependabot/bundler/file_fetcher_spec.rb

```diff
@@ -6,6 +6,26 @@
 require_common_spec "file_fetchers/shared_examples_for_file_fetchers"
 
 RSpec.describe Dependabot::Bundler::FileFetcher do
+  before do
+    stub_request(:get, File.join(url, ".ruby-version?ref=sha"))
+      .with(headers: { "Authorization" => "token token" })
+      .to_return(
+        status: 200,
+        body: fixture("github", "ruby_version_content.json"),
+        headers: { "content-type" => "application/json" }
+      )
+
+    stub_request(:get, File.join(url, ".tool-versions?ref=sha"))
+      .with(headers: { "Authorization" => "token token" })
+      .to_return(
+        status: 200,
+        body: fixture("github", "tool_versions_content.json"),
+        headers: { "content-type" => "application/json" }
+      )
+  end
+
+  before { allow(file_fetcher_instance).to receive(:commit).and_return("sha") }
```

Comment:
We could combine this setup into the preceding `before` block. (Bonus: helps us start paying down [RSpec/ScatteredSetup](https://docs.rubocop.org/rubocop-rspec/cops_rspec.html#rspecscatteredsetup))